api/types/types.go

```diff
@@ -737,7 +737,7 @@ type NetworksPruneReport struct {
 // on the creation of a new secret.
 type SecretCreateResponse struct {
 	// ID is the id of the created secret.
-	ID string
+	ID string `json:"ID"`
```

Comment:
figured it was a good idea to make this casing explicit - since the json case insensitivity is what caused the problem in the first place 😅